Packs/McAfee_ESM/Integrations/McAfee_ESM_v2/McAfee_ESM_v2.py

```diff
@@ -656,11 +657,14 @@ def __alarms_to_incidents(self, start_time: str, start_id: int = 0, limit: int =
         _, _, all_alarms = self.fetch_alarms(start_time=start_time, end_time=current_time, raw=True)
         all_alarms = filtering_incidents(all_alarms, start_id=start_id, limit=limit)
         if all_alarms:
-            current_run['time'] = all_alarms[0].get('triggeredDate', current_time)
+            current_run['time'] = all_alarms[0].get('triggeredDate', start_time)
```

Comment:
why is there a default value to triggeredDate? isn't it guaranteed to have it?
asking because if the alarm won't have this field we will keep using start_time in the next run and there will be duplicate incidents